responder/ext/cli.py

```diff
@@ -0,0 +1,127 @@
+"""
+Responder CLI.
+
+A web framework for Python.
+
+Commands:
+  run     Start the application server
+  build   Build frontend assets using npm
```

Comment:
@kennethreitz: Now after learning it, I think the second subcommand `responder build` is debatable. Did you keep and/or evolve it within Dyne, @tabotkevin?